gprofiler/metadata/metadata_collector.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+import time
+from typing import Dict, Optional, Union
+
+from gprofiler import __version__
+from gprofiler.metadata.cloud_metadata import get_static_cloud_instance_metadata
+from gprofiler.metadata.metadata_type import Metadata
+from gprofiler.metadata.system_metadata import get_static_system_info
+
+
+def get_static_metadata(spawn_time: Optional[float], run_args: Dict[str, Union[bool, str, int]] = None) -> Metadata:
+    if spawn_time is None:
+        spawn_time = time.time()
+    formatted_spawn_time = datetime.datetime.utcfromtimestamp(spawn_time).replace(microsecond=0).isoformat()
+    static_system_metadata = get_static_system_info()
+    cloud_metadata = get_static_cloud_instance_metadata()
+
+    metadata_dict: Metadata = {
+        "cloud_provider": cloud_metadata.pop("provider") if cloud_metadata is not None else "unknown",
+        "agent_version": __version__,
+        "spawn_time": formatted_spawn_time,
+    }
+    metadata_dict.update(static_system_metadata.__dict__)
+    if cloud_metadata is not None:
+        metadata_dict["cloud_info_wrapped"] = cloud_metadata
```

Comment:
Let's talk about this one IRL

Reply:
@leonid-granulate I'm changing to `cloud_info` instead of `cloud_info_wrapped`